modules/nw-multus-delete-network.adoc

```diff
@@ -39,13 +39,12 @@ metadata:
 spec:
   additionalNetworks: [] <1>
 ----
-<1>  If you are removing the configuration mapping for the only additional
-network attachment definition in the `additionalNetworks` collection, you must
+<1>  If you are removing the configuration mapping for the only additional secondary network attachment definition in the `additionalNetworks` collection, you must
```

Comment:
This way is only used to delete an additional network attachment created with the Cluster Network Operator, NOT from YAML file.